### PR TITLE
Add Tor (needed for fetch testing .onion domains)

### PR DESCRIPTION
**Pull request by ghost**
@Hainish I am currently working on fetch testing for .onion domains. This is needed for it to work.

### CODE REVIEW

**Comment by Bisaloo** [comment]
I would do

```sh
echo "deb http://deb.torproject.org/torproject.org $(lsb_release -sc) main"
```

to avoid issues when Xenial stops being Ubuntu LTS.

**Comment by ghost** [comment]
Thanks @Bisaloo.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 FROM ubuntu:latest
 MAINTAINER William Budington "bill@eff.org"
 
+RUN echo "deb http://deb.torproject.org/torproject.org $(lsb_release -sc) main" > /etc/apt/sources.list.d/tor.list && \
+  echo "deb-src http://deb.torproject.org/torproject.org $(lsb_release -sc) main" >> /etc/apt/sources.list.d/tor.list
```
Comment (Hainish) [inline Dockerfile]: `lsb_release` is not found.  Please test this locally first by running

```bash
sudo docker build -t electronicfrontierfoundation/https-everywhere-docker-base .
```

within your local repository.

**Comment by ghost** [comment]
@Hainish Try now. Can't test locally.

**Comment by Hainish** [comment]
@koops76 anything you can't test locally shouldn't be put into a PR.  Reviewers don't have the time to go through the process of helping contributors with every snafu that might come up.  A PR that you're seeking review on should be in a polished state that is at least functional in your own testing.

**Comment by ghost** [comment]
@Hainish I agree but I'm on Windows 10 right now, and I'm not sure how to test it in Bash on Ubuntu on Windows.

**Comment by Hainish** [comment]
@koops76 I think the best way is to get yourself set up with a Debian or Ubuntu VM for testing.  This is an older instructional but it should be still pretty accurate: http://www.psychocats.net/ubuntu/virtualbox

**Comment by ghost** [comment]
@Hainish Does this PR look good? Also do not use `ubuntu:latest` since it may change to a different version without notice, breaking many things including Tor Project PPA.

**Comment by Hainish** [comment]
@koops76 have you tested this locally?

**Comment by ghost** [comment]
@Hainish You can test it, I will only be able to test it tomorrow, staying up late for HTTPS Everywhere sake. 😉

**Comment by ghost** [comment]
@Hainish Tested locally, ready for review.

**Comment by Hainish** [comment]
Squashed and merged in https://github.com/EFForg/https-everywhere-docker-base/commit/cb32566949748e0913be5397a9427fb94b522461

**Pull request closed without merging**